pkgs/development/ocaml-modules/spacetime_lib/default.nix

```diff
@@ -1,4 +1,7 @@
-{ lib, fetchFromGitHub, buildDunePackage, owee }:
+{ lib, fetchFromGitHub, buildDunePackage, ocaml, owee }:
+
+lib.throwIfNot (lib.versionAtLeast "4.12" ocaml.version)
```

Comment:
Wouldn't this be possible using `minimalOCamlVersion`?

Reply:
No. Minimal sets a lower bound. Here it’s a upper bound.

Reply:
Right, I feel like `lib.versionOlder ocaml.version "4.13"` would be a bit more readable, but not too important.